pubsublite/driver/driver.go

```diff
@@ -0,0 +1,9 @@
+package driver
+
+import "context"
+
+type Pubsubber interface {
+	Publish(ctx context.Context, topic string, msg []byte) error
+	Subscribe(ctx context.Context, callback func(msg []byte)) error
```

Comment:
Although I like callbacks, we should offer options here as our Go client does. Channels seem to be the Go thing to do, so possibly something like:

https://github.com/nats-io/go-nats/blob/master/nats.go#L2721

Reply:
We've generally been avoiding exposing channels in the APIs of the Go CDK. 
See https://github.com/sourcegraph/gophercon-2018-liveblog/issues/35 for background.
What would be some advantages of adding them here?

Reply:
Channels seem polarizing, so will leave up to you.

Reply:
Okay, leaving that out for now. We can add it later if the need becomes clear.